rotkehlchen/api/v1/schemas.py

```diff
@@ -616,6 +616,10 @@ class HistoryEventSchema(DBPaginationSchema, DBOrderBySchema):
         SerializableEnumField(enum_class=HistoryBaseEntryType),
         load_default=None,
     )
+    excluded_entry_types = DelimitedOrNormalList(
+        SerializableEnumField(enum_class=HistoryBaseEntryType),
```

Comment:
I don't think it will make a difference, because it is a nested field where the output field has the load_default=None so if nothing is given, the field will be None. entry_types which has identical behaviour doesn't use it. I think it would be redundant.